cmd/gokr-packer/packer.go

```diff
@@ -713,6 +801,30 @@ func logic() error {
 		fromLiteral: *httpsPort,
 	})
 
+	for pkg1, fs1 := range extraFiles {
+		// check against root fs
+		paths := getDuplication(root, fs1)
+		if len(paths) > 0 {
+			return fmt.Errorf("extra files of package %s collides with rootfs: %v", pkg1, paths)
```

Comment:
This line still says rootfs instead of “root file system”

Reply:
changed